.github/workflows/release.yaml

```diff
@@ -2,8 +2,17 @@ name: "[ruff] Release"
 
 on:
   workflow_dispatch:
-  release:
-    types: [ published ]
+    inputs:
+      tag:
+        description: "The version to tag, without the leading 'v'. If not present, a dry run will be made"
+        type: string
+      sha:
+        description: "Optionally, the full sha of the commit to be released"
+        type: string
```

Comment:
I don't think this is used (at least, as far as I can tell).

Reply:
Aren't we able to select a branch anyway in the UI?

Reply:
This can be useful in cases where you want to re-run the release workflow on a specific commit. But you're right. We could create a branch pointing to that specific commit instead.